packages/amplify-provider-awscloudformation/src/utils/override-skeleton-generator.ts

```diff
@@ -0,0 +1,57 @@
+import path from 'path';
+import fs from 'fs-extra';
+import { getPackageManager } from 'amplify-cli-core';
+import execa from 'execa';
+import { $TSContext } from 'amplify-cli-core';
+
+export const generateOverrideSkeleton = async (context: $TSContext): Promise<void> => {
+  let projectPath: string;
```

Comment:
```suggestion
  const projectPath: string;
```

Reply:
Removed the use of it based on sachscode@ recommendation above.